carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -573,7 +573,7 @@ lanelet::ConstLaneletOrAreas WMBroadcaster::getAffectedLaneletOrAreas(const cav_
     prev_pt.x += pt.x;
     prev_pt.y += pt.y;
 
-    ROS_DEBUG_STREAM("After conversion in Map frame: Point X "<< gf_pts.back().x() <<" After conversion: Point Y "<< gf_pts.back().y());
+    ROS_DEBUG_STREAM("After conversion: Point X "<< gf_pts.back().x() <<" After conversion: Point Y "<< gf_pts.back().y());
```

Comment:
Why were the more descriptive "in TCM frame" and "in Map frame" parts of these comments removed? Seems arbitrary and provides less information in the print line.